### PR TITLE
Add check to see if there's a wallet able to pay an amount

### PR DESCRIPTION
**Pull request by kajoseph**

### CODE REVIEW

**Comment by justinkook** [comment]
https://github.com/bitpay/copay/blob/master/src/pages/integrations/bitpay-card/bitpay-card-topup/bitpay-card-topup.ts#L147

**Comment by justinkook** [comment]
LGTM

Tested getWallets with `minFiatCurrency` flag

Screen Shot 2020-02-28 at 11 32 37 AM

Screen Shot 2020-02-28 at 11 32 47 AM

File: src/providers/profile/profile.ts [hunk 1]
```diff
@@ -1837,4 +1853,16 @@ export class ProfileProvider {
     });
     return keyIdIndex >= 0;
   }
+
+  // Checks to see if a wallet exists with minimim fiat amount's worth in it (to pay invoice, for example)
+  public hasWalletWithFunds(fiatAmount: number, fiatCurrency: string): boolean {
```
Comment (justinkook) [inline src/providers/profile/profile.ts]: Used for BitpayID debit card implementation

**Comment by gabrielbazan7** [comment]
@kajoseph please squash all commits into a signed one, and I think is ready to merge.

**Comment by kajoseph** [comment]
@gabrielbazan7 Done. Just need you to stamp it again. Thanks!

**Comment by gabrielbazan7** [comment]
mmm something goes wrong with the PR. Could you check the changes ?

**Comment by kajoseph** [comment]
@gabrielbazan7 The last squash had 53 file changes for some reason. I reverted and re-squashed to get only the changes I've made into the diff. Should be good to go.

**Pull request merged**